packages/ui/src/composites/wui-network-image/index.ts

```diff
@@ -0,0 +1,42 @@
+import { html, LitElement } from 'lit'
+import { customElement, property } from 'lit/decorators.js'
+import '../../components/wui-icon'
+import '../../components/wui-image'
+import { resetStyles } from '../../utils/ThemeUtil'
+import styles from './styles'
+
+@customElement('wui-network-image')
+export class WuiNetworkImage extends LitElement {
+  public static styles = [resetStyles, styles]
+
+  // -- State & Properties -------------------------------- //
+
+  @property() public networkName = ''
```

Comment:
```suggestion
  @property() public name = 'unknown'
```

No need to prefix with network as composite name already indicates that